setup/usersWorkorders.js

```diff
@@ -0,0 +1,173 @@
+'use strict';
+
+const _ = require('lodash');
+const Promise = require('bluebird');
+const rp = require('request-promise');
+const login = require('../util/login');
+const recordUtils = require('../util/generate_record.js');
+const requestBodyUtils = require('../util/sync_request_bodies');
+const argv = require('yargs')
+      .reset()
+      .option('app', {
+        demand: true,
+        type: 'string',
+        alias: 'a',
+        describe: 'Cloud app base URL to target'
+      })
+      .option('username', {
+        demand: true,
+        type: 'string',
+        alias: 'u',
+        describe: 'Username to use to login to the app'
+      })
+      .option('password', {
+        demand: true,
+        type: 'string',
+        alias: 'p',
+        describe: 'Password to use to login to the app'
+      })
+      .option('numUsers', {
+        demand: true,
+        type: 'number',
+        alias: 'n',
+        describe: 'The number of users/workorders needed'
+      })
+      .option('concurrency', {
+        demand: true,
+        type: 'number',
+        alias: 'c',
+        describe: 'The concurrency at which to create resources'
+      })
+      .argv;
+
+function makeUser(id) {
+  return {
+    "password": "123",
+    "name" : "Jim Loader",
+    "phone" : "(999) 999 9999",
+    "username" : `loaduser${id}`,
+    "group": "Syl1GdSS", // Drivers
+    "position" : "Senior Load Tester",
+    "email" : `loaduser${id}@example.com`,
+    "notes" : "There certainly are a lot of Jim's around here...",
+    "banner" : "https://www.walldevil.com/wallpapers/w01/awesome-face-meme-wallpaper.jpg",
+    "avatar" : "https://upload.wikimedia.org/wikipedia/commons/thumb/f/fb/718smiley.svg/2000px-718smiley.svg.png"
+  };
+}
+
+function postUsers(sessionToken, numUsers, concurrency) {
+  const users = _.times(numUsers, makeUser);
+  const sessionRequest = rp.defaults({
+    json: true,
+    headers: requestBodyUtils.getSyncRequestHeaders({'X-FH-sessionToken': sessionToken})
+  });
+
+  process.stdout.write(`Creating users at a concurrency of ${concurrency}`);
```

Comment:
Almost! :) It's so that the newline character doesn't get appended, and then progress can be indicated simply by appending a '.' for each user that's created. I think it gives useful output :)

There are some `console.log` calls in other places here, so I clearly wasn't paying attention to the warnings :P